app/Http/Controllers/Api/v1/EntityTypeApiController.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Http\Controllers\Api\v1;
+
+use App\Models\EntityType;
+use App\Http\Resources\EntityTypeResource as Resource;
+
+class EntitytypeApiController extends ApiController
+{
+    /**
+     * @return \Illuminate\Http\Resources\Json\AnonymousResourceCollection
+     * @throws \Illuminate\Auth\Access\AuthorizationException
+     */
+    public function index()
+    {
+        return Resource::collection(EntityType::all());
```

Comment:
Please paginate the results in case we for one stupid reason end up with > 45 types on day :-)